generator/integration_tests/golden/v1/internal/golden_kitchen_sink_sources.cc

```diff
@@ -0,0 +1,39 @@
+// Copyright 2022 Google LLC
```

Comment:
nit: as discussed.

Reply:
I have a follow up PR to fix the copyright years. I am just going to merge this PR.

The right thing to do is to
- send the follow up to fix the copyright year in the GAPICs
- wait for it to merge
- rebase this PR
- regenerate the files
- merge this PR

... but who has time for that.

Reply:
#14226